rdrf/rdrf/services/io/defs/exporter.py

```diff
@@ -662,6 +663,24 @@ def _get_patient_stage_rules(self):
             data.append(rule_dict)
         return data
 
+    def _get_group_permissions(self):
+        data = []
+        for group in Group.objects.all():
+            permissions = []
+            for permission in group.permissions.all():
+                permission_dict = {
+                    "name": permission.name,
+                    "codename": permission.codename,
```

Comment:
To make forms readonly, trrf uses a per-group permission such as `Form 'myCommunication' is readonly (MND)`.

The `codename` for such permissions looks like so: `form_15_is_readonly`.

The PK of forms isn't going to be consistent across imports, so this PR will probably need to change those readonly permissions to use the form `name` rather than the `pk` as the identifier.

Then it will either need to add a migration to change these values in all registries, or we can change them manually.

Reply:
Hopefully I've address this in my latest commit.